service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EhrExtractMapper.java

```diff
@@ -63,6 +69,14 @@ public EhrExtractTemplateParameters mapBundleToEhrFhirExtractParams(
         return ehrExtractTemplateParameters;
     }
 
+    private String getPatientNhsNumber(GetGpcStructuredTaskDefinition getGpcStructuredTaskDefinition) {
```

Comment:
Could pass nhs number

Reply:
no, in that case what are you "getting" since it will appear that the "get" value is provided by the parameter? `getPatientNhsNumber("1234567890")` seems strange